GDJS/Runtime/runtimeobject.js

```diff
@@ -1107,7 +1170,7 @@ gdjs.RuntimeObject.prototype.separateFromObjects = function(objects, ignoreTouch
 
    //Check if their is a collision with each object
    for(var i = 0, len = objects.length;i<len;++i) {
-       if ( objects[i].id != this.id ) {
+       if ( objects[i].id != this.getUniqueId() ) {
```

Comment:
- Might be a micro optimization, but I prefer to keep `this._id` instead of `this.getUniqueId()` to avoid paying the cost to calling a function so often. In compiled language, such a simple function might be inlined/optimized out and hopefully that's also done by JS engines, but I would rather use `this._id` to be sure.

Reply:
That may be weird but it seems like using a getter function is faster: https://jsperf.com/getter-performance

Reply:
Well these are a micro benchmark :) 
I would be cautious about inferring anything from them because the difference seems very very small and they might not apply in a larger context where the JS engine has the optimize dozens of object types, passed through functions etc... at the end a simple property has more chance to be seen as directly addressable by the JS engine rather that something hiding behind functions.

Anyway, it's unlikely to make a change. :) I could be ok with this but let's not do this for now in this PR.